src/strings.json

```diff
@@ -59,6 +59,7 @@
   "whyPaused.exception": "Paused on exception",
   "whyPaused.resumeLimit": "Paused while stepping",
   "whyPaused.pauseOnDOMEvents": "Paused on event listener",
+  "whyPaused.breakpointConditionThrown": "Error with conditional breakpoint",
```

Comment:
can you also add this to https://github.com/devtools-html/debugger.html/blob/master/assets/locales/debugger.properties as well